src/Data/Audience.elm

```diff
@@ -36,6 +59,56 @@ type alias Audience =
 
 
 
+-- Decoder
+
+
+decodeDataAudience : Decode.Decoder (List Audience)
+decodeDataAudience =
+    Decode.at [ "data" ] (Decode.list decodeAudience)
+
+
+decodeAudience : Decode.Decoder Audience
+decodeAudience =
+    Decode.succeed Audience
+        |> required "id" Decode.int
+        |> required "name" Decode.string
+        |> required "type" (Decode.string |> Decode.andThen decodeType_)
+        |> required "folder" (Decode.nullable int)
+
+
+decodeType_ : String -> Decoder AudienceType
+decodeType_ val =
+    case val of
+        "curated" ->
+            Decode.succeed Curated
+
+        "shared" ->
+            Decode.succeed Shared
+
+        "user" ->
+            Decode.succeed Authored
+
+        _ ->
+            Decode.fail "unsupported type"
+
+
+
+--view
+
+
+view : Audience -> Html msg
+view audience =
+    div
+        [ class "audience"
+        ]
+        [ text (trim audience.name 60)
```

Comment:
I was wondering: "why is he not using the `String.trim` function?", and then I realised that you are trying to shorten the longer names... not a good idea to do this in the Elm side (IMHO), better to leave the complete name and handle from the CSS part the shortening into the ellipsis (this is what we extensively use in our codebase) 😉